### PR TITLE
Wire branding to designer

### PR DESCRIPTION
**Pull request by Tj-Tracy**
This PR removes the hard coded branding and replaces it with data coming from designer. This also removes the hard coded sidebar references. Examples: 

image

image

### CODE REVIEW

File: src/components/masters/program/ProgramPage.jsx [hunk 1]
```diff
@@ -95,11 +95,12 @@ class ProgramPage extends Component {
                     <Hero
                       programTitle={programName}
                       organizationLogo={{
-                        url: 'https://www.edx.org/sites/default/files/school/image/logo/gtx-logo-200x101.png',
-                        alt: 'Georgia Tech Institute of Technology logo',
+                        url: `${process.env.DESIGNER_BASE_URL}${programBranding.organization_logo.url}`,
```
Comment (tuchfarber) [inline src/components/masters/program/ProgramPage.jsx]: `organization_logo.url` is going to be absolute URL (at least in prod). 
Stage response:
```
"organization_logo": {
    "url": "https://stage-edx-designer.s3.amazonaws.com:443/original_images/gt_logo.png",
    "alt": ""
},
```

File: src/components/masters/program/ProgramPage.jsx [hunk 2]
```diff
@@ -95,11 +95,12 @@ class ProgramPage extends Component {
...
+                        alt: programBranding.organization_logo.alt,
                       }}
-                      textureImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt-cyber-title_bg_img_440x400.jpg"
-                      coverImage="https://prod-discovery.edx-cdn.org/media/degree_marketing/campus_images/gt_cyber_campus_image_1000x400.jpg"
+                      textureImage={`${process.env.DESIGNER_BASE_URL}${programBranding.texture_image}`}
+                      coverImage={`${process.env.DESIGNER_BASE_URL}${programBranding.cover_image}`}
```
Comment (tuchfarber) [inline src/components/masters/program/ProgramPage.jsx]: same as above for `textureImage` and `coverImage`. Both will be absolute URLs. (They will be coming from S3 not the backend)

File: src/components/masters/program/ProgramPage.jsx [hunk 3]
```diff
@@ -135,7 +141,16 @@ ProgramPage.propTypes = {
   pageContext: PropTypes.shape({
     programName: PropTypes.string.isRequired,
     programSlug: PropTypes.string.isRequired,
-    programUUID: PropTypes.string.isRequired,
+    programUUID: PropTypes.shape({
```
Comment (tuchfarber) [inline src/components/masters/program/ProgramPage.jsx]: I think you switched `programUUID` and `programBranding`?

Comment (Tj-Tracy) [reply src/components/masters/program/ProgramPage.jsx]: 👍 Fixed it and added a shape to branding.

File: src/components/masters/program/ProgramPage.jsx [hunk 4]
```diff
@@ -137,12 +143,22 @@ ProgramPage.propTypes = {
     programSlug: PropTypes.string.isRequired,
     programUUID: PropTypes.string.isRequired,
     programDocuments: PropTypes.shape({
-      display: PropTypes.bool,
-      header: PropTypes.string,
-      documents: PropTypes.arrayOf(PropTypes.shape({
-        display_text: PropTypes.string,
-        document: PropTypes.string,
-      })),
+      cover_image: PropTypes.string,
```
Comment (tuchfarber) [inline src/components/masters/program/ProgramPage.jsx]: You still have the branding schema inside the documents shape.

Comment (Tj-Tracy) [reply src/components/masters/program/ProgramPage.jsx]: 😅 I guess I did it too fast... should be fixed now

**Comment by tuchfarber** [comment]
Don't forget to squash :)

**Pull request merged**